app/views/volunteers/_profile.html.erb

```diff
@@ -1,35 +1,70 @@
-<div>
-    <div>
-        <h4>My Profile</h4>
-    </div>
-    <div>
-        <img src="<%= @volunteer.picture %>" height=50px>
-        <h5>Name:</h5> <p><%= @volunteer.first_name %> <%= @volunteer.last_name %></p>
-        <h5>Email:</h5> <p><%= @volunteer.email %></p>
-        <h5>Phone:</h5> <p><%= @volunteer.mobile_phone %></p>
-        <h5>Location:</h5> <p><%= @volunteer.city %>, <%= @volunteer.state %> </h5>
-    </div>
-    <div>
-        <h4>Personal Information</h4>
-    </div>
-    <div>
-        <h5>Birthdate:</h5> <p><%= @volunteer.birthdate %></p>
-        <h5>Gender:</h5> <p><%= @volunteer.gender %></p>
-        <h5>Employer:</h5> <p><%= @volunteer.employer %></p>
-        <h5>Position:</h5> <p><%= @volunteer.position %></p>
-    </div>
-    <div>
-        <h5>Availability:</h5> <p><%= @volunteer.availability %></p>
-        <h5>My Skills:</h5><p><%= @volunteer.skill %></p>
-        <h5>Notes regarding my skills and availability:</h5>
-        <p><%= @volunteer.volunteer_notes %></p>
+<style>
+ 
+    .body {
+        margin-top: 250px;
+        margin-left: 25px;
+        margin-right: 25px;
+    }
+    
+    h4 {
+        color: #7ac043;
+    }
+    
+    .picture {
+        width: 100%;
+        height: auto;
+    }
+</style>
+
+<div class='body'>
+
+    <h4>My Profile</h4>
+    
+    <div class="card mb-3" style="max-width: 540px;">
+        <div class="row no-gutters">
+            <div class="col-md-6">
+                <% if @volunteer.picture? %>
+                        <%= image_tag @volunteer.picture.url, class: "picture" %>
+                <% end %>
+            </div>
+            <div class="col-md-6">
+                <div class="card-body">
+                <p class="card-title"><b>Name:</b> <%= @volunteer.first_name %> <%= @volunteer.last_name %></p>
+                <p class="card-text"><b>Email:</b> <%= @volunteer.email %></p>
+                <p class="card-text"><b>Phone:</b> <%= @volunteer.mobile_phone %></p>
+                <p class="card-text"><b>Location:</b> <%= @volunteer.city %>, <%= @volunteer.state %></p>
+                </div>
+            </div>
+        </div>
     </div>
-    <div>
-       <%= link_to 'Edit Profile', edit_volunteer_path(@volunteer) %>
-        <!-- Do we want to add a link to delete a profile? Or ask them to submit a request to delete the profile to the administrator -->
+            
+    <h4>Personal Information</h4>
+    
+    <div class="card mb-3" style="max-width: 540px;">
+        <div class="row no-gutters">
+            <div class="col-md-12">
+                <div class="card-body">
+                    <p class="card-text"><b>Birthdate:</b> <%= @volunteer.birthdate %></p>
+                    <p class="card-text"><b>Employer:</b> <%= @volunteer.employer %></p>
+                    <p class="card-text"><b>Gender:</b> <%= @volunteer.gender %></p>
+                    <p class="card-text"><b>Position:</b> <%= @volunteer.position %></p>
```

Comment:
These seem a bit out of order. Consider reordering to
1. Gender
2. Birthdate
3. Employer
4. Position